CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+### How to contribute:
+
+It's recommended that you first contact ctsm-software@ucar.edu to discuss the changes or additions that you propose.
+Work with them to think about the design and implemention of the changes as well as the planning
+and scheduling for the change to come into CTSM.
+
+It's also recommened that you join the ctsm-dev@ucar.edu google groups in order 
+to be informed about the science that is happening in CTSM as well as the latest develepments and tags.
+The ctsm-dev group is here:
+
+https://groups.google.com/a/ucar.edu/forum/#!forum/ctsm-dev
+
+Use the help from the WIKI below to setup a fork and personal branch in github to put your developments
+on and keep up to date with the main development CTSM. Once the changes are sufficiently advanced you
+can form a Pull Request on github under
+
+https://github.com/ESCOMP/ctsm/pulls
```

Comment:
Is this really the way to make a PR? When I went there, it didn't show me my branches on my personal fork. I think you need to create a PR from your fork.

Reply:
You can do either one. On your fork you can make a PR from your list of branches. On ESCOMP when you first make the PR you hit the "compare across forks" link at the top of the page. Then you tell it your fork and it'll have your list of branches after you do that.

Reply:
Ah, thanks for clarifying.